packages/open-payments/src/client/requests.ts

```diff
@@ -84,7 +85,13 @@ export const post = async <TRequest, TResponse>(
   const url = requestUrl.href
 
   try {
-    const { data, status } = await axiosInstance.post<TResponse>(url, body)
+    const { data, status } = await axiosInstance.post<TResponse>(url, body, {
+      headers: accessToken
+        ? {
+            Authorization: `GNAP ${accessToken}`
+          }
+        : {}
```

Comment:
was missing previously

Reply:
The `properly POSTs request` test is failing with the addition of `headers: {}`